docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+
+services:
+  rethinkdb:
+    container_name: rethinkdb
+    image: rethinkdb:2.3
+    restart: unless-stopped
+    volumes:
+        - "rethinkdb-data:/data"
+    command: "rethinkdb --bind all --cache-size 2048"
+
+  adb:
+    container_name: adb
+    image: sorccu/adb:latest
+    restart: unless-stopped
+    volumes: 
+      - "/dev/bus/usb:/dev/bus/usb"
+    privileged: true
+
+  stf:
+    container_name: stf
+    image: devicefarmer/stf
+    ports:
+      - "7100:7100"
+        #- "7105:7105"
+      - "7110:7110"
+        #- "7120:7120"
+      - "7400-7500:7400-7500"
+    environment:
+      - TZ='America/Los_Angeles'
+      - RETHINKDB_PORT_28015_TCP=tcp://rethinkdb:28015
+      - STF_ADMIN_EMAIL=YOUR_EMAIL
```

Comment:
I would use something more eye-catching like <YOUR EMAIL>

Reply:
Looks like your comment may have been cut off. Did you have something in particular to use as the placeholders?

Reply:
yeah, something went wrong when posting the comment.
I mean `<YOUR EMAIL>`.

Reply:
Updated!